Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,15 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+
+Excercise 1:
+
+1- Since we have n * n * n inside the while loop, this would be O(n^3)
+
+2- For this one I feel like it should be O(n^4) because of the 4 nested for loops
```

Comment:
Similar for this one, notice that the last loop (from k+1 to k+10) will always have the same amount of iterations regardless of n.